clients/v1/clients.proto

```diff
@@ -166,3 +166,41 @@ message IntrospectTokenResponse {
   // ExpiresAt indicates when the token expires.
   google.protobuf.Timestamp expires_at = 4;
 }
+
+service RolesService {
+  rpc RegisterRoleResources (stream RegisterRoleResourcesRequest) returns (RegisterRoleResourcesResponse) {
+    option (sams_required_scopes) = "sams::roles.resources::write";
```

Comment:
technically we should require write _and_ delete right, as this RPC can do both?